heft-plugins/heft-node-loc-plugin/src/schemas/node-loc-plugin.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Node Localization Configuration",
+  "description": "This schema describes the \"config/node-loc.json\" config file used to configure \"@rushstack/heft-node-loc-plugin\".",
+  "type": "object",
+
+  "additionalProperties": false,
+  "required": ["srcFolder", "generatedTsFolder"],
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+
+    "extends": {
+      "description": "Optionally specifies another JSON config file that this file extends from.  This provides a way for standard settings to be shared across multiple projects.",
+      "type": "string"
+    },
+
+    "srcFolder": {
+      "type": "string",
+      "description": "Source code root directory."
+    },
+
+    "generatedTsFolder": {
+      "type": "string",
+      "description": "Output directory for generated typings."
+    },
+
+    "stringsOutputFolders": {
```

Comment:
Phasing would be simpler if this plugin operated out of a folder other than `src/` and had a single output folder that was referenced via extra `../` in the import path. Emitting multiple `.json` files being emitted just because we have multiple module formats is bad for package sizes. Something to revisit with regards to static assets and the like, as well.